packages/table/test/formatsTests.tsx

```diff
@@ -69,7 +71,7 @@ describe("Formats", () => {
                 majority have never stirred?
             `;
 
-            const style = { height: "200px" };
+            const style = { height: "200px", position: "relative" };
```

Comment:
> if so, we should add this css rule to the TABLE_TRUNCATED_TEXT class instead of just the style on the test

that's what i was wondering. lemme test it out locally.